File: Dillio-Backend.DAL/Dillio-Backend.Entities/Core/Domain/Review.cs
using System;

namespace Dillio_Backend.BLL.Core.Domain
{
    public class Review
    {
        public int Id { get; set; }
        public string ReviewDescription { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public float Rating{ get; set; }
        public Product Product { get; set; }
        public int ProductId { get; set; }
        public string UserId { get; set; }
        public Store Store { get; set; }
        public int StoreId { get; set; }
        public DateTime ReviewDate{ get; set; }
        public ApplicationUser User { get; set; }

    }
}

File: dillio-app/src/app/_models/Review.ts

export interface Reviews{
    id?:number;
    description?:string;
    name?:string;
    email?:string;
    productId?:number;
    UserId?:number;
    StoreId:number;
    Date?:string;
    rating?:number;
    // avatarUrl?:string;
}
export interface ReviewToDisplay{
  id?:number;
  reviewDescription?:string;
  name?:string;
  email?:string;
  UserId?:number;
  reviewDate?:string;
  rating?:number;
  // avatarUrl?:string;
}
export interface ReviewResolved {
  reviews: ReviewToDisplay[];
  error?: any;
}
